src/drivers/spi/spi.h

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file spi.h
+ * @brief
+ * @author Denis Deryugin <deryugin.denis@gmail.com>
+ * @version
+ * @date 03.12.2018
+ */
+
+#ifndef DRIVERS_SPI_H_
+#define DRIVERS_SPI_H_
+
+#include <drivers/char_dev.h>
+#include <util/macro.h>
+
+struct spi_ops;
+
+struct spi_device {
+	int    id;
+	int    cs;
+	int    flags;
+	char  *name;
+	struct dev_module *dev;
+	struct spi_ops *spi_ops;
+	void  *priv;
+};
+
+struct spi_ops {
+	int (*init)(struct spi_device *dev);
+	int (*select)(struct spi_device *dev, int cs);
+	int (*transfer)(struct spi_device *dev, uint8_t *in, uint8_t *out, int cnt);
+};
+
+extern struct spi_device *spi_dev_by_id(int id);
+extern int spi_transfer(struct spi_device *dev, uint8_t *in, uint8_t *out, int cnt);
+extern int spi_transfer(struct spi_device *dev, uint8_t *in, uint8_t *out, int cnt);
```

Comment:
Two identical declarations of `spi_transfer`

Reply:
Fixed.